Model/Quote/QuoteAction/Currency.php

```diff
@@ -39,19 +40,20 @@ public function getActionData(CartInterface $cart): array
 
         return [
             [
-                'type' => self::SET_CURRENCY,
+                'type' => self::SET_DISPLAY_CURRENCY,
                 'data' => [
                     'currency' => $cartCurrency->getQuoteCurrencyCode(),
                     'rate' => $cart->getCurrency()->getBaseToQuoteRate(),
                     'format_string' => $format,
                 ],
             ],
-            [
-                'type' => self::SET_GATEWAY_CURRENCY,
-                'data' => [
-                    'currency' => $cartCurrency->getQuoteCurrencyCode(),
-                ],
-            ],
+            // Removing for now. More work is needing to be done to get this to work
```

Comment:
Question: What is the impact of removing this?

Reply:
Nothing. It doesn't work atm